pandas/tests/indexes/interval/test_setops.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+import pytest
+
+from pandas import Index, IntervalIndex, Timestamp, interval_range
+import pandas.util.testing as tm
+
+
+@pytest.fixture(scope='class', params=[None, 'foo'])
+def name(request):
+    return request.param
+
+
+def monotonic_index(start, end, dtype='int64', closed='right'):
+    return IntervalIndex.from_breaks(np.arange(start, end, dtype=dtype),
+                                     closed=closed)
+
+
+def empty_index(dtype='int64', closed='right'):
+    return IntervalIndex(np.array([], dtype=dtype), closed=closed)
+
+
+class TestIntervalIndex:
+
+    @pytest.mark.parametrize("sort", [None, False])
```

Comment:
could move this to a fixture as well (or we *may* already have this defined in a higher level conftest)